ui/src/containers/CreateNodeForm.js

```diff
@@ -0,0 +1,229 @@
+import {
+  createNodeAction,
+  clearCreateNodeErrorAction
+} from '../ducks/app/nodes';
+
+const CreateNodeLayout = styled.div`
+  height: 100%;
+  overflow: auto;
+  padding: ${padding.larger};
+`;
+
+const PageHeader = styled.h2`
+  margin-top: 0;
+`;
+
+const CreateNodeFormContainer = styled.div`
+  input {
+    padding: ${padding.small};
+    font-size: ${fontSize.large};
+    width: 250px;
+    display: block;
+    border-radius: 4px;
+    border: 1px solid ${gray};
+  }
+
+  input:focus {
+    border-color: #007eff;
+    box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075),
+      0 0 0 3px rgba(0, 126, 255, 0.1);
+    outline: none;
+  }
+`;
+
+const InputFeedback = ({ error }) =>
+  error ? <div className="input-feedback">{error}</div> : null;
+
+const Label = ({ error, className, children, ...props }) => {
+  return (
+    <label className="label" {...props}>
+      {children}
+    </label>
+  );
+};
+
+const TextInput = ({
+  type,
+  id,
+  label,
+  error,
+  value,
+  onChange,
+  className,
+  ...props
+}) => {
+  const classes = classnames(
+    'input-group',
+    {
+      'animated shake error': !!error
+    },
+    className
+  );
+  return (
+    <CreateNodeFormContainer className={classes}>
+      <Label htmlFor={id} error={error}>
+        {label}
+      </Label>
+      <DebounceInput
+        minLength={1}
+        debounceTimeout={300}
+        id={id}
+        className="text-input"
+        type={type}
+        value={value}
+        onChange={onChange ? onChange : null}
+        {...props}
+      />
+      <InputFeedback error={error} />
+    </CreateNodeFormContainer>
+  );
+};
+
+const ActionContainer = styled.div`
+  margin: ${padding.base} 0;
+`;
+
+const ErrorMessage = styled.span`
+  margin-top: ${padding.base};
+  color: red;
+`;
+
+const initialValues = {
+  name: '',
+  ssh_user: '',
+  hostName_ip: '',
+  ssh_port: '',
+  ssh_key_path: '',
+  sudo_required: false,
+  workload_plane: true,
+  control_plane: false
+};
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string().required(),
+  ssh_user: Yup.string().required(),
+  hostName_ip: Yup.string().required(),
+  ssh_port: Yup.string().required(),
+  ssh_key_path: Yup.string().required(),
+  sudo_required: Yup.boolean().required(),
+  workload_plane: Yup.boolean().required(),
+  control_plane: Yup.boolean().required()
+});
+
+class CreateNodeForm extends React.Component {
+  componentWillUnmount() {
+    this.props.clearCreateNodeError();
+  }
+
+  render() {
+    const { intl, errors } = this.props;
+    return (
+      <CreateNodeLayout>
+        <PageHeader>Create a New Node</PageHeader>
+        <Formik
+          initialValues={initialValues}
+          validationSchema={validationSchema}
+          validateOnBlur={false}
+          validateOnChange={false}
+          onSubmit={this.props.createNode}
+        >
+          {props => {
+            const { values, handleChange } = props;
+            return (
+              <Form>
```

Comment:
please render the form as we defined in the wireframe (except for the checkbox style)

Reply:
The idea was to use the same Input as Login page. We already discuss about this issue.